xtask/src/commands/release/execute_plan.rs

```diff
@@ -140,10 +147,11 @@ fn make_git_changes(dry_run: bool, release_plan_str: &str, release_plan: &Plan)
     if dry_run {
         println!("Dry run: would commit changes to branch: {branch_name}");
     } else {
+        Command::new("git").arg("add").arg(".").status().context("Failed to stage changes")?;
```

Comment:
This makes me slightly uncomfortable (we should pass in the list of new files instead), but it's not a big deal given we have to through pass a few eyes.